VideoWeb/VideoWeb/ClientApp/src/app/services/media-stream.service.ts

```diff
@@ -42,7 +42,7 @@ export class MediaStreamService {
         return from(this.navigator.mediaDevices.getUserMedia({ video: { deviceId: { exact: device.deviceId } } })).pipe(
             map(stream => {
                 const cloneStream = stream.clone();
```

Comment:
I know this isn't part of this particular change but could we just return the stream rather than cloning it if we aren't going to filter it? Or is this needed either way?

Reply:
I gave it a go without cloning and it seems to be okay. Updated the tests so it no longer expects clone to be called too